src/standalone-js/src/index.js

```diff
@@ -0,0 +1,13 @@
+/* global window */
+import * as lumaGL from 'luma.gl';
+import * as deckGLCore from 'deck.gl/core';
+import * as deckGLLayers from 'deck.gl/core-layers';
+
+import Deck from './deck';
+
+if (typeof window !== 'undefined') {
+  window.deck = Object.assign({}, window.deck, deckGLCore, deckGLLayers, {Deck});
```

Comment:
Nit: Don't completely hide `Deck`...

Reply:
`_Deck`?

Reply:
OK, though my preference would be that the new wrapper component has a separate name. It could be called `DeckGL` or `DeckLite` or `DeckGLLite` or similar.